MarineSABRES_SES_Shiny/.claude/skills/pre-deploy/SKILL.md

```
@@ -0,0 +1,117 @@
+
+```bash
+git describe --tags --abbrev=0 2>/dev/null
+```
+
+Warn if the version in DESCRIPTION doesn't match or exceed the latest tag.
+
+### 8. Deployment checklist walkthrough
+
+Present the key checklist items from `deployment/DEPLOYMENT_CHECKLIST.md` as a summary:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)